Tests/QuranAudioKitTests/QuranAudioRequestBulderTests.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mohammad Abdurraafay on 2023-02-11.
+//
+
+import Foundation
+import XCTest
+@testable import QuranAudioKit
+import QuranKit
+
+class GappedAudioRequestBuilderTests: XCTestCase {
+    private var audioRequestBuilder: QuranAudioRequestBuilder!
+    private var reciter: Reciter!
+    
+    override func setUpWithError() throws {
+        reciter = .gappedReciter
+        audioRequestBuilder = GappedAudioRequestBuilder()
+    }
+    
+    override func tearDown() async throws {
+        audioRequestBuilder = nil
+        reciter = nil
+    }
+    
+    func testAudioRequestContainsBismillahWhenThePlaybackIsNotRepeated() throws {
+        let expectation = expectation(description: "waiting for promise to fulfill")
+        let from = try XCTUnwrap(AyahNumber(sura: .secondSura, ayah: Sura.firstSura.firstVerse.ayah))
+        let to = try XCTUnwrap(AyahNumber(sura: .secondSura, ayah: Sura.firstSura.firstVerse.ayah))
+        let totalNumberOfAyah = 1
```

Comment:
Shouldn't this be 2 instead? The basmalah + the first verse of the second sura.

Reply:
We're only requesting one ayah to be recited, hence it should be 1.